CMakeLists.txt

```diff
@@ -547,15 +547,15 @@ endif()
 
 # Add 'check' target - quick test
 if (NOT MFEM_USE_MPI)
-  add_custom_target(check
+  add_custom_target(mfem_check
```

Comment:
This breaks the intended target name when building MFEM on its own.

Maybe a better way to do this is to use a variable-defined name, e.g. `MFEM_CHECK_TARGET`, with a default value `check` that can be overwritten before including MFEM as a sub-project. What do you think @ajkunen?